cartofacade/carto_facade_test.go

```diff
@@ -0,0 +1,127 @@
+package cartofacade_test
```

Comment:
I'm ok with merging this in with just these minimal tests for now, but by the time the project is done we should have test cases for every cartofacade api call that ends up calling the cgo api.